### PR TITLE
Badges, sign in pages and profile modal window

### PR DESCRIPTION
**Pull request by IraKhomyk**

### CODE REVIEW

**Review by ksarkisyan — CHANGES_REQUESTED** [review]

File: ProjectFrontEnd/src/app/app.component.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
 <mat-sidenav-container class="container">
-    <app-header></app-header>
```
[the remark this answers is not in the document]
Comment (IraKhomyk) [reply ProjectFrontEnd/src/app/app.component.html:2]: fixed

File: ProjectFrontEnd/src/app/app.component.ts [hunk 2]
```diff
@@ -5,6 +5,6 @@ import { Component } from '@angular/core';
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-
 export class AppComponent {
+  title = 'CMFCTN';
```
[the remark this answers is not in the document]
Comment (IraKhomyk) [reply ProjectFrontEnd/src/app/app.component.ts]: fixed

File: ProjectFrontEnd/src/app/modules/dashboard/components/request-achievement/request-achievement.component.html [hunk 3]
```diff
@@ -10,17 +10,21 @@
   <span mat-dialog-title>
     Please, leave a message and your achievement will be assigned soon
   </span>
-
-  <div class="body-request">
-    <mat-select [(value)]="achievement">
-      <mat-option *ngFor="let item of [].constructor(3)" [value]="achievement">{{achievement}}</mat-option>
+  
+  <form [formGroup]="requestForm" class="body-request">
+    <mat-select [(value)]="selectAchievement" id="achievement" formControlName="achievement" required>
```
[the remark this answers is not in the document]
Comment (IraKhomyk) [reply ProjectFrontEnd/src/app/modules/dashboard/components/request-achievement/request-achievement.component.html]: fixed

File: ProjectFrontEnd/src/app/shared/edit-profile/edit-profile-modal-win/edit-profile-modal-win.component.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+})
+export class EditProfileModalWinComponent {
+  user: userShortInfo = {
+    firstName: 'Ira',
+    lastName: 'Khomyk',
+    photoUrl: './../../../../assets/myphoto.jpg',
+    color: '',
+  }
+
+  profileForm: FormGroup = new FormGroup({
```
[the remark this answers is not in the document]
Comment (IraKhomyk) [reply ProjectFrontEnd/src/app/shared/edit-profile/edit-profile-modal-win/edit-profile-modal-win.component.ts:22]: fixed

File: ProjectFrontEnd/src/app/shared/edit-profile/edit-profile-modal-win/edit-profile-modal-win.component.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+                {{user | initials}}
+            </div>
+        </div>
+    </div>
+
+    <form [formGroup]="profileForm">
+        <div>
+            <mat-form-field class="example-full-width">
+                <mat-label class="text-uppercase">first name</mat-label>
+                <textarea matInput id="firstName" formControlName="firstName" required></textarea>
```
[the remark this answers is not in the document]
Comment (IraKhomyk) [reply ProjectFrontEnd/src/app/shared/edit-profile/edit-profile-modal-win/edit-profile-modal-win.component.html]: fixed

File: ProjectFrontEnd/src/styles.scss [hunk 6]
```diff
@@ -93,6 +93,11 @@ body {
   font-size: 14px;
 }
 
+.mat-form-field.mat-form-field-invalid .mat-form-field-label.mat-accent,
+.mat-form-field.mat-form-field-invalid .mat-form-field-label .mat-form-field-required-marker {
+  display: none;
+}
+
 //------------------------------request achievement modal window------------
 .request-model-container .mat-dialog-container {
```
[the remark this answers is not in the document]
Comment (IraKhomyk) [reply ProjectFrontEnd/src/styles.scss:102]: fixed

File: ProjectFrontEnd/src/index.html [hunk 7]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>FrontEndProject</title>
+  <title>Camification</title>
```
[the remark this answers is not in the document]
Comment (IraKhomyk) [reply ProjectFrontEnd/src/index.html]: fixed

**Review by ksarkisyan — APPROVED** [review]

**Pull request merged**